### DISCUSSION TITLE
Script Kit for Linux - Developer Preview

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#669] [Announcements]
# Script Kit for Linux - Developer Preview

## Why "Developer Preview"

1. Building from the "Mac" source

Currently, the Linux build builds from the exact same branch as the Mac build. While this works fine, for now, we'll want two separate release cadences and feature sets. This could be done through a branch, but then I'd have to set up a new release server. I'll figure it out details next year.

2. The Linux build is missing all the OS-specific tools

Linux currently doesn't support `getSelectedText`, `getTabs`, and other utils that are written in AppleScript. I'm planning to tackle many of those utils in Rust, which theoretically should allow them to be cross-platform, but that's another task for next year.

3. I've Only Tested on Ubuntu through a Parallels vm

Obviously will need some more real-world testing.

## Where to Download

Download the AppImage here:
https://github.com/johnlindquist/kitapp/releases/tag/v1.5.1

**Comment by canvural** [comment]
Tested on Ubuntu 21.10 And it worked ok. Some small issues but I guess it's expected for now.

Thank you for this!
